flow/aggregates.py

```diff
@@ -282,8 +276,8 @@ def __init__(self, aggregator, project):
 
         # We need to register the aggregates for this instance using the
         # project provided.
-        self._aggregates = list()
-        self._aggregate_ids = dict()
+        self._aggregates = []
+        self._aggregate_ids = {}
```

Comment:
Initialize empty lists/dicts with object literals. It's faster and more Pythonic.